sass/pages/check-mail/_check_mail.scss

```diff
@@ -0,0 +1,56 @@
+.check-body{
+	background-color: #F7F7F7;
+
+	.check-section{
+		background-color: $white;
+		text-align: center;
+
+		.check-div{
+			width: 30%;
+			@include center(true,true);
+			background-color: $white;
+
+			.check-form{
+				
+				.check-field{
+					border-radius: 7.5px;
+					border: none;
+					-webkit-box-shadow: 0px 0px 10px 2px rgba(184,175,184,0.51);
```

Comment:
If you're using Koala, you don't need to add every prefix, just activate autoprefixer checkbox.